tests/conftest.py

```diff
@@ -119,9 +139,13 @@ def strategy(
         angle_stable_master,
         san_token_gauge,
         pool_manager,
+        deploy_strategy_proxy.address
     )
     strategy.setKeeper(keeper, {"from": gov})
     vault.addStrategy(strategy, 10_000, 0, 2 ** 256 - 1, 1_000, {"from": gov})
+    deploy_strategy_proxy.approveStrategy(san_token_gauge, strategy.address, {"from": gov})
```

Comment:
you don't need to use '.address' for it to automatically pull the address

Reply:
Yeah, I know. Still I do it because it makes it easier to read for me but removing them